dowhy/causal_estimator.py

```diff
@@ -126,6 +182,48 @@ def do(self, x):
     def construct_symbolic_estimator(self, estimand):
         raise NotImplementedError
 
+    def get_confidence_intervals(self, estimate):
```

Comment:
better to call it singular "interval" rather than "intervals", since there is only one interval returned.